_workinggroups/softwaredevelopertools.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Software/Developer Tools 
+layout: plain
+redirect_from:
+  - /activities/softwaredevelopertools.html
+---
+
+The Software/Developer Tools Working Group aims to help HEP community in developing the latest and modern feature-rich projects
+with the help of software tools such as code editors, static/dynamic code analyzers, compilers, debuggers, performance analyzers and so on.
```

Comment:
Suggest "aid" instead of "help" to avoid having help twice in the same sentence.

Reply:
Done.